dali/base/dafdesc.cpp

```diff
@@ -3659,6 +3659,20 @@ bool getDefaultSpillPlane(StringBuffer &ret)
 #endif
 }
 
+bool getDefaultIndexBuildStoragePlane(StringBuffer &ret)
+{
+#ifdef _CONTAINERIZED
+    if (getGlobalConfigSP()->getProp("storage/@indexBuildPlane", ret))
```

Comment:
@jakesmith the order doesn't seem quite right here.  This means the property set on storage takes precedence over a setting on the component.

Reply:
you're right, will change.

Reply:
@ghalliday - please see new commit.